src/builder.rs

```diff
@@ -658,9 +682,34 @@ impl Builder {
             self.bundle_type,
             self.spends,
             self.outputs,
-            self.burn,
+            SpecificBuilderParams::BundleParams(self.burn),
         )
     }
+
+    /// Builds an action group containing the given spent and output notes.
+    ///
+    /// The returned action group will have no proof or signatures; these can be applied with
+    /// [`ActionGroup::create_proof`] and [`ActionGroup::apply_signatures`] respectively.
+    pub fn build_action_group<V: TryFrom<i64>>(
+        self,
+        rng: impl RngCore,
+        timelimit: u32,
+    ) -> Result<ActionGroup<InProgress<Unproven<OrchardZSA>, Unauthorized>, V>, BuildError> {
+        if !self.burn.is_empty() {
+            return Err(BuildError::BurnNotEmptyInActionGroup);
+        }
+        let action_group = bundle(
+            rng,
+            self.anchor,
+            self.bundle_type,
+            self.spends,
+            self.outputs,
+            SpecificBuilderParams::ActionGroupParams(self.reference_notes),
+        )?
+        .unwrap()
+        .0;
```

Comment:
Actually, we need metadata as well, like in normal build

Reply:
Done